objects/src/accounts/data.rs

```diff
@@ -23,16 +23,20 @@ use super::{
 ///
 /// The intent of this struct is to provide an easy way to serialize and deserialize all
 /// account-related data as a single unit (e.g., to/from files).
-#[derive(Debug, PartialEq, Eq, Clone)]
+#[derive(Debug, Clone)]
 pub struct AccountData {
     pub account: Account,
     pub account_seed: Option<Word>,
-    pub auth: AuthData,
+    pub auth_secret_key: AuthSecretKey,
```

Comment:
Do we still need the `AuthData` enum defined on line 98 below?

Reply:
Removed it